rpc/src/module/chain.rs

```diff
@@ -44,7 +44,7 @@ pub trait ChainRpc {
     ) -> Result<Vec<CellOutputWithOutPoint>>;
 
     #[rpc(name = "get_live_cell")]
-    fn get_live_cell(&self, _out_point: OutPoint) -> Result<CellWithStatus>;
+    fn get_live_cell(&self, _out_point: OutPoint, _with_data: bool) -> Result<CellWithStatus>;
```

Comment:
change `with_data` to optional param:

```suggestion
    fn get_live_cell(&self, _out_point: OutPoint, _with_data: Option<bool>) -> Result<CellWithStatus>;
```

Reply:
I think it's better explicit here, if we add another more argument in the future, it will make user confused.